src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -144,23 +142,31 @@ internal static ScriptHostConfiguration CreateScriptHostConfiguration(WebHostSet
                 LoggerFactoryBuilder = settings.LoggerFactoryBuilder
             };
 
+            if (inStandbyMode)
+            {
+                scriptHostConfig.FileLoggingMode = FileLoggingMode.Always;
+                scriptHostConfig.HostConfig.StorageConnectionString = null;
+                scriptHostConfig.HostConfig.DashboardConnectionString = null;
+                scriptHostConfig.RootScriptPath = Path.Combine(Path.GetTempPath(), "Functions", "Standby");
```

Comment:
When in standby mode, we write our placeholder functions to a temp directory that we point the placeholder host at. The host will start up normally on those files.